### ISSUE TITLE
Readonly disappears forever if window is resized while not readonly

### ISSUE THREAD

**Issue by markfinn** [roroettg/ngx-blockly#78]
If anything calls `Blockly.svgResize` while readonly is false you will never see the readonly workspace again unless you call `Blockly.svgResize` again.  `Blockly.svgResize` is called a lot in my code for dynamic reflows, but it's also easy to trigger in NGX-Blockly itself by resizing the browser window.

The cause is that when the `secondaryContainer` div is hidden it will report a size of 0x0 and `Blockly.svgResize` will smash the blockly workspace down to that invisible size.  That's fine while hidden, but when you un-hide, the SVG workspace is still 0x0.  The fix is to call `svgResize` after removing the `hidden` class from `secondaryContainer`:

````diff
     public setReadonly(readOnly: boolean) {
         this.readOnly = readOnly;
         if (readOnly) {
             this.secondaryContainer.nativeElement.classList.remove('hidden');
             if (!this._secondaryWorkspace) {
                 const config = {...this.config};
                 config.readOnly = true;
                 this._secondaryWorkspace = Blockly.inject(this.secondaryContainer.nativeElement, config);
             }
             Blockly.Xml.clearWorkspaceAndLoadFromXml(Blockly.Xml.textToDom(this.toXml()), this._secondaryWorkspace);
+            Blockly.svgResize(this._secondaryWorkspace);
         } else {
             if (this._secondaryWorkspace) {
                 this.secondaryContainer.nativeElement.classList.add('hidden');
````

Note that this fix alone actually doesn't work yet because of a [Blockly issue](https://github.com/google/blockly/issues/5930) but I have a fix requested over there too.  At the moment I have both this change and that one monkey-patched in and everything works as expected when the widows is resized in either state.

Thanks!

**Comment by markfinn** [comment]
Bump to keep from being stale.

**Comment by markfinn** [comment]
the fix to blockly is merged at google/blockly#5997, just need this half.

**Comment by markfinn** [comment]
bump to keep from being stale.
